connect/src/main/protobuf/spark/connect/expressions.proto

```diff
@@ -155,4 +156,7 @@ message Expression {
     string expression = 1;
   }
 
+  // represent * (e.g. SELECT *)
+  message UnresolvedStar {
+  }
```

Comment:
We need to expand this message later to support full fields of the star:
```
case class UnresolvedStar(target: Option[Seq[String]]) extends Star with Unevaluable {

```

which is will be used to support `Struct`.

Reply:
one example is `structDf.select(xxhash64($"a", $"record.*")`